### ISSUE TITLE
[Technical story] Choose a theme.

### ISSUE THREAD

**Issue by ccamara** [cadaveresinmob/c_inmobiliarios#18]
Currently we are using the same theme as the wordpress installation ([Twenty fifteen](https://www.drupal.org/project/twentyfifteen)), but that theme may be inappropriate:

Pros:
* Same theme as wordpress (same visual style)
* Responsive menu

Cons:
* Too narrow
* Only one sidebar

This task depends on the wireframes (Issue #17) 

Reference:
* [Themes' list for drupal 7](https://www.drupal.org/project/project_theme?f[0]=&f[1]=&f[2]=drupal_core%3A103&f[3]=sm_field_project_type%3Afull&text=&solrsort=iss_project_release_usage+desc&op=Search)

**Comment by ccamara** [comment]
Put on hold till we define wireframes (issue #17 )

**Comment by numeroteca** [comment]
To add more info to the conversation: Twentyfifteen was use because it was the default theme in Wordpress when we created the site. I'd suggest to use the default theme in drupal (or other not so narrow) to have a better look to the editing form.

**Comment by ccamara** [comment]
@numeroteca your comment made me realize that the site was not working as I thought it was. 

Drupal has (or can have) two themes: the one used in the frontend and another for the backend. The administration theme is only displayed for administration pages (drupal considers an admin page everything that comes after `/admin` path) and is it possible to display it when creating or editing content, which was the configuration I used. The problem is that I didn't give permissions to anyone other than administrators to see the admin theme, and since I am an admin user I didn't realize about that problem, hence the unexpected behaviour.

In commit cb377c5 I have given anyone permissions to use administration theme, so we would have a workaround here: we may still have some nice theme to display till we decide which theme are we going to use and at the same time editions will be easier, as administration theme uses the whole page width.

**Comment by numeroteca** [comment]
FYI: I am still seeing the old narrow twentyfifteen while editng.

**Comment by ccamara** [comment]
I'm afraid you may be checking on the wrong site: since @skotperez created a new server ( #19 ) I am deploying changes into http://new.cadaveresinmobiliarios.org

**Comment by ccamara** [comment]
After internal discussions and as @numeroteca stated in issue #17 ([see comment here](https://github.com/cadaveresinmob/c_inmobiliarios/issues/17#issuecomment-229922304) we have decided to use bootstrap.

I'm closing this issue as the theme has been chosen and has been implemented. We will be creating new issues should we need to tweak it further.

A site using bootstrap subtheme can be seen in http://cinmobiliarios.carloscamara.es
